pkg/tinydtls/Makefile.include

```diff
@@ -56,3 +59,48 @@ endif
 ifneq (,$(filter tinydtls_ecc,$(USEMODULE)))
   DIRS += $(PKG_BUILDDIR)/ecc
 endif
+
+# Translate 'CONFIG_' options to package specific flags. This checks if the
+# option is being set via Kconfig or CFLAGS
+
+ifneq (,$(filter -DDTLS_PSK,$(CFLAGS)))
+  # For backwards compability. This can be removed after release 2020.10
+  $(warning Warning! DTLS_PSK is deprecated use CONFIG_DTLS_PSK)
+  CFLAGS += -DCONFIG_DTLS_PSK
+endif
+ifneq (,$(or $(CONFIG_DTLS_PSK),$(filter -DCONFIG_DTLS_PSK,$(CFLAGS))))
+    CFLAGS += -DDTLS_PSK
+endif
+
+ifneq (,$(filter -DDTLS_ECC,$(CFLAGS)))
+  # For backwards compability. This can be removed after release 2020.10
+  $(warning Warning! DTLS_ECC is deprecated use CONFIG_DTLS_ECC)
+  CFLAGS += -DCONFIG_DTLS_ECC
+endif
+ifneq (,$(or $(CONFIG_DTLS_ECC),$(filter -DCONFIG_DTLS_ECC,$(CFLAGS))))
+    CFLAGS += -DDTLS_ECC
+endif
+
+ifneq (,$(filter -DTINYDTLS_DEBUG,$(CFLAGS)))
+  # For backwards compability. This can be removed after release 2020.10
+  $(warning Warning! TINYDTLS_DEBUG is deprecated use CONFIG_TINYDTLS_DEBUG)
+  CFLAGS += -DCONFIG_TINYDTLS_DEBUG
+endif
+ifneq (,$(or $(CONFIG_TINYDTLS_DEBUG),$(filter -DCONFIG_TINYDTLS_DEBUG,$(CFLAGS))))
+    CFLAGS += -DTINYDTLS_DEBUG
```

Comment:
Maybe unify with the code above which sets the log? (also it checks for `TINYDTLS_LOG`)

Reply:
fixed.

Reply:
Regarding `TINYDTLS_LOG`, see #13290. `TINYDTLS_LOG_LVL` can be set directly instead if required.